docs/api/web-contents.md

```diff
@@ -588,6 +588,15 @@ Emitted when media starts playing.
 
 Emitted when media is paused or done playing.
 
+#### Event: 'audio-state-changed'
+
+Returns:
+
+* `event` Event
+* `audible` boolean - Whether audio is currently playing.
```

Comment:
This feels a little imprecise; it might be good to copy the spirit of the Chromium comments to help guide readers through the use case of children

```diff
- `audible` boolean - Whether audio is currently playing.
+ `audible` boolean - true if one or more frames or child contents are emitting audio.
```